src/cli/rustup_mode.rs

```diff
@@ -1598,9 +1598,15 @@ fn doc(cfg: &Cfg, m: &ArgMatches) -> Result<utils::ExitCode> {
     let doc_url = if let Some(topic) = m.get_one::<String>("topic") {
         topical_path = topical_doc::local_path(&toolchain.doc_path("").unwrap(), topic)?;
         topical_path.to_str().unwrap()
-    } else if let Some((_, _, path)) = DOCS_DATA.iter().find(|(name, _, _)| m.get_flag(name)) {
+    } else if let Some((name, _, path)) = DOCS_DATA.iter().find(|(name, _, _)| m.get_flag(name)) {
+        writeln!(
+            process().stdout().lock(),
+            "Opening docs -{} in your browser",
```

Comment:
The `-` prefix looks strange to me, what's your motivation for adding this?

Reply:
Used it to represent name of package

Reply:
Sure, but why use a `-` prefix for it?

Reply:
Not much thought went into it. Happy to change

Reply:
Was trying to stay with what the user put in command line